internal/config/notify/help.go

```diff
@@ -262,6 +262,18 @@ var (
 			Optional:    true,
 			Type:        "sentence",
 		},
+		config.HelpKV{
+			Key:         target.KafkaCompressionCodec,
+			Description: "specify compression_codec of the kafka",
+			Optional:    true,
+			Type:        "none|snappy|gzip|lz4|zstd",
+		},
+		config.HelpKV{
+			Key:         target.KafkaCompressionLevel,
+			Description: "specify compression level of the kafka cluster",
```

Comment:
%s/specify compression level of the kafka cluster/specify compression level of the Kafka cluster/g